Makefile

```diff
@@ -1,6 +1,9 @@
-RELEASE_BRANCH=release
```

Comment:
This seems unrelated, remove?

Reply:
Moved to a separate PR: https://github.com/acm19/aws-request-signing-apache-interceptor/pull/72